dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -0,0 +1,1097 @@
+                "  order by arrivalDelay ";
+        Row[] result = snc.sql(query).collect();
+        Log.getLogWriter().info("SS - Qyery executed successfully and query result is ::" + ((GenericRow) result[0]).get(0));
+    }
+
+    /**
+     * This method retrives the trend in arrival delays across all airlines in the US
+     */
+
+    public static void HydraTask_avgArrDelayWithGroupByOrderByForTrendAnalysis() {
+        String query = "select AVG(ArrDelay) ArrivalDelay, yeari\n" +
+                "  from airline \n" +
+                "  group by yeari \n" +
+                "  order by yeari ";
+        Row[] result = snc.sql(query).collect();
+//        Long expectedValue = Long.parseLong("6.735443");
+//        Assert.assertEquals(expectedValue, ((GenericRow) result[0]).get(0));
+        Log.getLogWriter().info("SS - Qyery executed successfully and query result is ::" + ((GenericRow) result[0]).get(0));
+    }
+
+    /**
+     * This method retrives Which airline out of SanFrancisco had most delays due to weather
+     */
+
+    public static void HydraTask_sumWeatherDelayWithGroupByWithLimit() {
+        String query = "SELECT sum(WeatherDelay) totalWeatherDelay, airlineref.DESCRIPTION \n" +
+                "  FROM airline, airlineref \n" +
+                "  WHERE airline.UniqueCarrier = airlineref.CODE AND  Origin like '%SFO%' AND WeatherDelay > 0 \n" +
+                "  GROUP BY DESCRIPTION \n" +
+                "  limit 20 ";
+        Row[] result = snc.sql(query).collect();
+        Log.getLogWriter().info("SS - Qyery executed successfully and query result is ::" + ((GenericRow) result[0]).get(0));
+    }
+
+    /**
+     * This method updates the description for a code in row table. Suppose a particular Airline company say 'Delta Air Lines Inc.' re-brands itself as 'Delta America'
+     * --- the airline code can be updated in the row table
+     */
+
+    public static void HydraTask_updateRowTable() {
+        String query1 = "UPDATE AIRLINEREF SET DESCRIPTION='Delta America' WHERE CODE='DL' ";
+        Row[] result = snc.sql(query1).collect();
+        Log.getLogWriter().info("SS - Qyery executed successfully");
+        String query2 = "SELECT * FROM AIRLINEREF WHERE CAST(CODE AS VARCHAR(25))='DL'";
+        Row[] result1 = snc.sql(query2).collect();
+//        Long expectedValue = Long.parseLong("DL");
+//        Assert.assertEquals(expectedValue, ((GenericRow) result[0]).get(0));
+        Log.getLogWriter().info("SS - Qyery executed successfully and query result is ::" + ((GenericRow) result1[0]).get(0));
+    }
+
+    protected String getDataLocation() {
+        String scriptPath = null;
+        try {
+            hd = getHostDescription();
+            scriptPath = hd.getGemFireHome() + sep + ".." + sep + "snappy"
+                    + sep + "quickstart" + sep + "data";
+            Log.getLogWriter().info("SS - scriptPath is ::" + scriptPath);
+        } catch (Exception e) {
```

Comment:
change the method description protected String getDataLocation() throws Exception and remove the  and remove the catch method as anyways it is doing nothing